<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Handball Manager</string>
  </property>
  <widget class="QLabel" name="lblClubName">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>10</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zagreb</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNextOpponent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next Opponent:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblPlayers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>700</width>
     <height>800</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Player</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Age</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Position</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Attack</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Defense</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Selected</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="cbAtkGk">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>350</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAtkCb">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>300</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAtkLb">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>290</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAtkRb">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>290</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAtkLw">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>220</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAtkRw">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>220</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAtkP">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>250</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblAttack">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>150</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Attack Tactics</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartMatch">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>850</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start Match</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDefRb">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>550</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDefRw">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>590</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDefCb">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>510</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDefLw">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>590</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDefP">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>550</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDefense">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>450</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Defense Tactics</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDefLb">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>550</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDefGk">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>610</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmSelection">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>770</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Selection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
